docs/protocol/port-forwarding.md

```diff
@@ -75,7 +75,7 @@ The actual forwarding process will vary based on your router, below is the gener
 2. Advanced Settings > Port Forwarding
 3. Within the port forwarding screen we will see the following fields, all fields have been filled accordingly to our defaults, except for the Computer IP Address, you will replace this with the computer IP address you received in the first steps.
     1. **Computer IP Address:** `192.168.0.25`
-    2. **Protocol:** `TCP`
+    2. **Protocols:** `TCP`, `UDP`
```

Comment:
Not this line, but on line 15 it feels like we should do:

opened up your firewall for TCP traffic on port 30333 with the following command `sudo ufw allow 30333/tcp`
->
opened up your firewall for traffic on port 30333 with the following command `sudo ufw allow 30333/tcp` for TCP and `sudo ufw allow 30333/udp` for UDP

Or maybe just 

opened up your firewall for TCP & UDP traffic on port 30333 with the following command `sudo ufw allow 30333`

Reply:
I like the first, more elaborate option. Good point Jim, added 👍